dojo/importers/importer/importer.py

```diff
@@ -19,18 +19,20 @@
 import base64
 import logging
 
+from dojo.tools.factory import get_parser
+
 logger = logging.getLogger(__name__)
 deduplicationLogger = logging.getLogger("dojo.specific-loggers.deduplication")
 
 
 class DojoDefaultImporter(object):
 
-    def create_test(self, scan_type, engagement, lead, environment, tags=None,
+    def create_test(self, scan_type, test_type_str, engagement, lead, environment, tags=None,
```

Comment:
`test_type_name` would be a better name of that parameter

Reply:
Changed.